index.bs

```diff
@@ -248,6 +248,37 @@ or <dfn for=stream>bidirectional</dfn>.
  </tbody>
 </table>
 
+[=WebTransport stream=] has the following events:
+
+<table class="data" dfn-for="stream-event">
+ <thead>
+  <tr>
+   <th>event
+   <th>definition
+   <th>incoming
+   <th>outgoing
+   <th>bidirectional
```

Comment:
I'm confused by the terms used here; a bidirectional stream can be either incoming or outgoing.  Did you mean to use send/receive stream?

Reply:
Here is my mental model.

 - HTTP/3 stream is specified in HTTP/3.
 - WebTransport uses several HTTP/3 streams. They are specified in [WEB-TRANSPORT-HTTP3].
   - The CONNECT stream (one for a session)
   - Incoming streams (from the client PoV, ditto below)
   - Outgoing streams
   - Bidirectional streams
  - SendStreams and ReceiveStreams are JS objects. This specification specifies the behavior of them by using capabilities and events for [=WebTransport streams=].

This section consists of adapters of concepts defined in [WEB-TRANSPORT-HTTP3].

Does this make sense?

Reply:
I find it weird using "Incoming streams" to specifically mean "Incoming unidirectional streams", when we have both `incomingBidirectionalStreams` and `incomingUnidirectionalStreams` in the API.

Reply:
I can replace ["incoming"](https://w3c.github.io/webtransport/#stream-incoming) with "incoming unidirectional" and ["outgoing"](https://w3c.github.io/webtransport/#stream-outgoing) with "outgoing unidirectional", in a separate change.

Reply:
https://github.com/w3c/webtransport/pull/289